sql/moz-fx-data-shared-prod/monitor_frontend_derived/event_monitoring_live_v1/metadata.yaml

```diff
@@ -0,0 +1,20 @@
+friendly_name: Event Monitoring Live
```

Comment:
In cases where you're modifying only `workgroup_access`  it's not necessary to add any other generated metadata e.g. https://github.com/mozilla/bigquery-etl/blob/main/sql/moz-fx-data-shared-prod/org_mozilla_broken_site_report/user_reports/metadata.yaml